docs/displaysystem/angular.md

```diff
@@ -88,11 +82,130 @@ z.angularUnwatchGlobal(String name)
 
 ```
 
-Let's make an button, that increment 'run' variable by 1 when it is clicked.
-z.angularBind("run", 0) will initialize 'run' to zero. And then register watcher of 'run'.
+Let's make a button. When it is clicked, the value of `run` will be increased 1 by 1.
+
+<img src="/assets/themes/zeppelin/img/screenshots/display_angular2.png" width="60%" />
+
+`z.angularBind("run", 0)` will initialize `run` to zero. And then, it will be also applied to `run` in `z.angularWatch()`.
+After clicked button, you'll see both `run` and `numWatched` are increased by 1.
+
+<img src="/assets/themes/zeppelin/img/screenshots/display_angular3.png" width="60%" />
+
+## Let's Simpler and more Intuitive
```

Comment:
Hmm, do we have some amazing name for this?
Let's make it simpler and more intuitive
is ok